core/src/test/java/com/linecorp/armeria/common/stream/DefaultStreamMessageTest.java

```diff
@@ -56,7 +56,7 @@ class DefaultStreamMessageTest {
     void onSubscribeBeforeOnComplete() throws Exception {
         final BlockingQueue<String> queue = new LinkedTransferQueue<>();
         // Repeat to increase the chance of reproduction.
-        for (int i = 0; i < 8192; i++) {
+        for (int i = 0; i < 128; i++) {
```

Comment:
If this is a problem only on CI, can we use a lower value only when `CI` environment variable is present? (and also log?)

Reply:
Fixed. :-)